docs/pipelineruns.md

```diff
@@ -1392,17 +1392,18 @@ Your `PipelineRun`'s `status` field can contain the following fields:
 
 ### Configuring usage of `TaskRun` and `Run` embedded statuses
 
-Currently, the default behavior is for the statuses of `TaskRun`s and `Run`s within this `PipelineRun`
-to be embedded in the `status.taskRuns` and `status.runs` fields. This will change in the future to
-instead default to `status.childReferences` being populated with references to the `TaskRun`s and
+Currently, the default behavior is default to `status.childReferences` being populated with references to the `TaskRun`s and
```

Comment:
NIT: One "default" too much in"Currently, the default behavior is default to". Maybe 

> Currently, the default behavior is for the `status.childReferences` to be populated with... 

or 

> Currently, the default behavior is for populate `status.childReferences` with...